### ISSUE TITLE
Add wall clock time information to display

### ISSUE THREAD

**Issue by mpickering** [mpickering/eventlog2html#75]
Seconds since epoch is actually included in the eventlog.

```
Events:
79710: created capset 0 of type CapsetOsProcess
79770: created capset 1 of type CapsetClockDomain
80042: created cap 0
80083: assigned cap 0 to capset 0
80127: assigned cap 0 to capset 1
80414: capset 1: wall clock time 1562256742s 76420000ns (unix epoch)
80770: capset 0: pid 10499
81116: capset 0: parent pid 6213
82268: capset 0: RTS version "GHC-8.9.0.20190628 rts_p"
82423: capset 0: args: ["./Run","+RTS","-hb","-l-au"]
```

See `wall clock time 15622...`, that's it.

We should find out which event this is in and include the date the profile was taken in the visualisation.

**Comment by BinderDavid** [comment]
Do you mean the
```
WallClockTime 
    capset :: !Capset
    sec :: !Word64
    nsec :: !Word32
```
constructor in http://hackage.haskell.org/package/ghc-events-0.9.0/docs/GHC-RTS-Events.html ?

**Comment by BinderDavid** [comment]
I'll have a look at this later today. It shouldn't be too much work.

**Comment by mpickering** [comment]
Merged in 13fd77e27784b4e41d376571ede6315750a0b529
